about.html

```diff
@@ -0,0 +1,145 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link href='https://unpkg.com/boxicons@2.0.7/css/boxicons.min.css' rel='stylesheet'>
+    <link rel="stylesheet" href="style.css">
+    <title>About -  Capstone</title>
+</head>
+<body>
+    <header class="header">
+        <nav class="nav">
+            <div class="icon" id="hamburger">
+                <i class='bx bx-menu hamburger'></i>
+            </div>
+            <div class="logo">
+                <a href="index.html">Logo</a>
+            </div>
+            <ul class="top-nav">
+                <li class="top-nav__item">
+                    <a href="#" class="top-nav__link">
+                        <i class='bx bxl-twitter'></i>
+                    </a>
+                </li>
+                <li class="top-nav__item">
+                    <a href="#" class="top-nav__link">
+                        <i class='bx bxl-facebook'></i>
+                    </a>
+                </li>
+                <li class="top-nav__item">
+                    <a href="#" class="top-nav__link">English</a>
+                </li>
+                <li class="top-nav__item">
+                    <a href="#" class="top-nav__link">Logout</a>
+                </li>
+            </ul>
+            <ul class="nav-list">
+                <li class="nav-item">
+                    <div class="icon">
+                        <i class='bx bx-x close' id="close-icon"></i>
+                    </div>
+                </li>
+                <li class="nav-item"><a href="index.html" class="nav-link">Home</a></li>
+                <li class="nav-item"><a href="about.html" class="nav-link">About</a></li>
+            </ul>
+        </nav>
+        <div class="hero-container">
```

Comment:
As requested by the previous reviewer, It will be great to apply the design to this section as provided. This will make your project resemble the given design. ⬇️ 

![123692990-c5f73900-d857-11eb-9256-12a30714866b](https://user-images.githubusercontent.com/32361963/123927795-dd3c4080-d995-11eb-8757-d305a9842f71.png)